qforce/qm/xtb.py

```diff
@@ -0,0 +1,452 @@
+import os.path
+
+from colt import Colt
+import numpy as np
+from ase.units import Hartree, mol, kJ, Bohr
+#
+from .qm_base import WriteABC, ReadABC
+from ..elements import ATOM_SYM
+
+
+class xTB(Colt):
+    _user_input = """
+    charge_method = xtb :: str ::
+    """
+
+    _method = []
+
+    def __init__(self):
+        self.required_hessian_files = {'hess_file': ['hessian'],
+                                       'pc_file': ['charges'],
+                                       'wbo_file': ['wbo'],
+                                       'coord_file': ['xtbopt.xyz'], }
+        self.read = ReadxTB
+        self.write = WritexTB
+
+class WritexTB(WriteABC):
+    def hessian(self, file, job_name, config, coords, atnums):
+        """ Write the input file for hessian and charge calculation.
+
+        Parameters
+        ----------
+        file : file
+            The file object to write the command line.
+        job_name : string
+            The name of the job.
+        config : config
+            A configparser object with all the parameters.
+        coords : array
+            A coordinates array of shape (N,3), where N is the number of atoms.
+        atnums : list
+            A list of atom elements represented as atomic number.
+        """
+        name = file.name
+        base, filename = os.path.split(name)
+        # Given that the xTB input has to be given in the command line.
+        # We create the xTB command template here.
+        cmd = f'xtb {job_name}_input.xyz --ohess --chrg {config.charge} ' \
```

Comment:
I'm not a big fan of this. Why not just use the[ input file](https://github.com/grimme-lab/xtb/blob/main/man/xcontrol.7.adoc) (like you partly did) for the job settings so that the user can just call xtb as:

xtb coord_file --input settings_file

so we don't provide any bash scripts to the user by default, like other QM integrations

Reply:
@selimsami I literally asked this question months ago (https://github.com/grimme-lab/xtb/issues/560).
For xTB, the long term solution would be the xtb-python, so no one wish to do this in the (near) future. Before the xtb-python is matured, we have to stick to this.